modules/rosa-getting-started-install-configure-cli-tools.adoc

```diff
@@ -0,0 +1,159 @@
+// Module included in the following assemblies:
+//
+// * rosa_getting_started/rosa-getting-started.adoc
+
+[id="rosa-getting-started-install-configure-cli-tools_{context}"]
+= Installing and configuring the required CLI tools
+
+Use the following steps to install and configure the AWS, {product-title} (ROSA) and OpenShift CLI tools on your workstation.
+
+.Prerequisites
+
+* You have an AWS account.
+* You created a Red Hat account.
++
+[NOTE]
+====
+You can create a Red Hat account by navigating to link:https://console.redhat.com[console.redhat.com] and selecting *Register for a Red Hat account*.
+====
+
+.Procedure
+
+. Install and configure the latest AWS CLI (`aws`).
+.. Follow the link:https://aws.amazon.com/cli/[AWS Command Line Interface] documentation to install and configure the AWS CLI for your operating system.
++
+Specify your `aws_access_key_id`, `aws_secret_access_key`, and `region` in the `.aws/credentials` file. See link:https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html[AWS Configuration basics] in the AWS documentation.
++
+[NOTE]
+====
+You can alternatively use the `AWS_DEFAULT_REGION` environment variable to set the default AWS region.
+====
+.. Query the AWS API to verify if the AWS CLI is installed and configured correctly:
++
+[source,terminal]
+----
+$ aws sts get-caller-identity
+----
++
+.Example output
+[source,terminal]
+----
+<aws_account_id>    arn:aws:iam::<aws_account_id>:user/<username>  <aws_user_id>
+----
+
+. Install and configure the latest ROSA CLI (`rosa`).
+.. Navigate to the link:https://console.redhat.com/openshift/downloads[*Downloads*] page on the {OCM} and download the latest version of the `rosa` CLI for your operating system.
+.. Extract the `rosa` binary file from the downloaded archive. The following example extracts the binary from a Linux tar archive:
++
+[source,terminal]
+----
+$ tar xvf rosa-linux.tar.gz
+----
+.. Add `rosa` to your path. In the following example, the `/usr/local/bin` directory is included in the path of the user:
++
+[source,terminal]
+----
+$ sudo mv rosa /usr/local/bin/rosa
+----
+.. Verify if the `rosa` CLI tool is installed correctly by querying the `rosa` version:
++
+[source,terminal]
+----
+$ rosa version
+----
++
+.Example output
+[source,terminal]
+----
+1.1.7
+----
++
+.. Optional: Generate the command completion scripts for the `rosa` CLI. The following example generates the Bash completion scripts for a Linux machine:
++
+[source,terminal]
+----
+$ rosa completion bash | sudo tee /etc/bash_completion.d/rosa
+----
+.. Optional: Source the scripts to enable `rosa` command completion from your existing terminal. The following example sources the Bash completion scripts for `rosa` on a Linux machine:
```

Comment:
I'm not comfortable with "source" as a verb. Plus, it feels as if there's assumed knowledge about where the scripts are being sourced from.

Reply:
Thanks for highlighting this. I have subsequently applied this alternative:

`.. Optional: Enable `rosa` command completion from your existing terminal. The following example enables Bash completion for `rosa` in an existing terminal on a Linux machine:`